### PR TITLE
Survey form

### PR DESCRIPTION
**Pull request by 2004-okari**
This survey form represents different inputs used in every everyday workplace.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Survey</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
```
Comment (2004-okari) [inline index.html:10]: Kindly use semantic tags such as main and section to enhance Search engine optimisation.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    Web development technologies.
+                </textarea>
+            </div>
+            <div>
+                <button>
+                    <input id="submit" type="submit">
+                </button>
+            </div>
+        </form>
+    </div>
```
Comment (2004-okari) [inline index.html:67]: Remove divs covering inputs and set classes to the inputs to style them

File: index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Survey</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="layout">
+        <h1 id="title">kublwear product</h1>
+        <p id="description">To create a better and functioning world through providing safer energy producing machinery</p>
+        <form id="survey-form">
```
Comment (2004-okari) [inline index.html:14]: Add action and method attributes to the form element.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            <div>
+                <label id="name-label">Full Names</label>
+                <input id="name" type="text" placeholder="John Doe" required>
+                <label id="email-label">Enter email</label>
+                <input id="email" type="email" placeholder="user@gmail.com" required>
```
Comment (2004-okari) [inline index.html:19]: Please add value attributes to the inputs.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                <input id="number" type="number" min="16" max="80" value="age" placeholder="16">
+            </div>
+            <div>
+                <label>How often do you use our products?</label><br>
+                <input type="checkbox" id="vehicle1" name="vehicle2" value="once">
+                <label for="vehicle2"> rarely</label><br>
+                <input type="checkbox" id="vehicle1" name="vehicle3" value="Oftenly">
+                <label for="vehicle3"> once in a while</label><br>
+                <input type="checkbox" id="vehicle1" name="vehicle4" value="Occasionaly">
+                <label for="vehicle1">Occasionaly</label>
```
Comment (2004-okari) [inline index.html:30]: Kindly insert the checkboxes before the labels. It enhances visuality and professionality.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+            </div>
+            <div>
+                <label >would you recommend this products?</label><br>
+                <input  type="radio"  name="decision" value="y">
+                <label>Yes</label><br>
+                <input  type="radio"  name="decision" value="m">
+                <label>Maybe</label><br>
+                <input  type="radio"  name="decision" value="n">
+                <label>Never</label><br>
```
Comment (2004-okari) [inline index.html:39]: Kindly insert the radio inputs before the labels. To enhance readability and visuals.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+                <label>What important features are we missing?</label>
+                <input id="checkbox" class="text-bar" type="text">
+                <label>How could we improve our product to better meet your needs?</label>
+                <input id="checkbox" class="text-bar" type="text">
+            </div>
+            <div>
+                <label for="review">Review:</label>
+                <textarea id="review" name="review" rows="5" cols="40">
+                    Web development technologies.
+                </textarea>
```
Comment (2004-okari) [inline index.html:59]: Add placeholder text and value attributes to enhance professionalism.

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+    font-size: 20px;
+    border: 1px solid gray;
+    padding: 10px;
+
+}
+
+#submit {
+    background-color: aqua;
+    border: none;
+}
```
Comment (2004-okari) [inline styles.css:93]: Use % for container widths and heights to enhance website responsiveness.

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,93 @@
+body {
```
Comment (2004-okari) [inline styles.css:1]: Could you use factors of 16 as your font size to enhance readability, visuals and professionalism?

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+    margin: 20px 60px;
+    background-color: antiquewhite;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 20px;
+}
+
+label {
+    font-size: larger;
```
Comment (2004-okari) [inline styles.css:16]: Use an actual size measurement to enhance professionalism.

**Review by 2004-okari — COMMENTED** [review]
Good job so far. You are almost there👍👍

**Review by Cabraham1 — APPROVED** [review]
Hi @2004-okari  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**